spec/factories/teams.rb

```diff
@@ -0,0 +1,18 @@
+# == Schema Information
+#
+# Table name: teams
+#
+#  id         :bigint           not null, primary key
+#  name       :string(100)      not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+# Indexes
+#
+#  index_teams_on_name  (name) UNIQUE
+#
+FactoryBot.define do
+  factory :team do
+    name { Faker::Name.name }
```

Comment:
To avoid duplicate names coming from Faker, which can cause confusing failures when this factory is called multiple times: `Faker::Name.unique`